### PR TITLE
Review data otherway

### PR DESCRIPTION
**Pull request by ghost**
Working user with review with refresh

### CODE REVIEW

File: app/src/main/java/com/example/watchnext/fragments/users/AddReviewFragment.java [hunk 1]
```diff
@@ -68,7 +71,11 @@ private void setPostButtonOnClickListener() {
     }
 
     private void post(View view) {
-        // TODO
+        Review r = new Review(titleEditText.getText().toString(), descriptionEditText.getText().toString());
+        Model.instance.addReview(() -> {
+            Log.d("nana", "nana");
```
Comment (ghost) [inline app/src/main/java/com/example/watchnext/fragments/users/AddReviewFragment.java]: Remove log

File: app/src/main/java/com/example/watchnext/fragments/users/FeedFragment.java [hunk 2]
```diff
@@ -33,25 +39,47 @@
...
         reviewList.setLayoutManager(new LinearLayoutManager(getContext()));
         reviewListAdapter = new ReviewListAdapter();
         reviewList.setAdapter(reviewListAdapter);
         reviewListAdapter.setOnItemClickListener((v, position) -> {
+            String reviewId = Objects.requireNonNull(reviewWithOwnerListViewModel.getData().getValue()).get(position).review.getId();
+            // TODO: get review id to next
             Navigation.findNavController(v).navigate(FeedFragmentDirections.actionFeedFragmentToReviewDetailsFragment());
         });
-
+        reviewWithOwnerListViewModel.getData().observe(getViewLifecycleOwner(), list1 -> refresh());
```
Comment (ghost) [inline app/src/main/java/com/example/watchnext/fragments/users/FeedFragment.java]: Change list1 to reviewWithOwnerList

File: app/src/main/java/com/example/watchnext/models/firebase/reviews/ReviewModelFirebase.java [hunk 3]
```diff
@@ -33,13 +33,13 @@ public ReviewModelFirebase() {
 
     public void getAllReviews(long since, GetAllReviewsListener listener) {
         db.collection(COLLECTION_NAME)
-                .whereGreaterThanOrEqualTo(Review.LAST_UPDATED, new Timestamp(since, 0))
+                .whereGreaterThanOrEqualTo("updateDate", new Timestamp(since, 0))
```
Comment (bsuro10) [inline app/src/main/java/com/example/watchnext/models/firebase/reviews/ReviewModelFirebase.java]: Why not const?

File: app/src/main/java/com/example/watchnext/models/firebase/reviews/ReviewModelFirebase.java [hunk 4]
```diff
@@ -33,13 +33,13 @@ public ReviewModelFirebase() {
...
                 .get()
                 .addOnCompleteListener(task -> {
                     List<Review> list = new LinkedList<>();
```
Comment (bsuro10) [inline app/src/main/java/com/example/watchnext/models/firebase/reviews/ReviewModelFirebase.java]: List<Review> list = new ArrayList<>()
this is more standard

**Review by bsuro10 — COMMENTED** [review]
Great Job

**Pull request merged**